infra/Dockerfile.keycloak

```diff
@@ -0,0 +1,24 @@
+FROM quay.io/keycloak/keycloak:26.0 AS builder
```

Comment:
Is this Dockerfile from the docs? If so, link it?

Reply:
yes based onhttps://www.keycloak.org/server/containers will link